src/components/Cart.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+    import {cart, totalPrice} from "../store/cart"
+    import {
+        Button,
+        Modal,
+        ModalHeader,
+        ModalBody,
+        ModalFooter,
+        Table,
+        Input
+    } from 'sveltestrap';
+
+    let isOpen = false;
+    const toggle = () => (isOpen = !isOpen);
+
+    const checkout = () => {
+        alert('checkout!')
+        cart.deleteAll()
+        toggle()
+    }
+
+    $: $cart.map((item, index) => {
+        if (item.quantity === 0) {
+            $cart.splice(index, 1)
+        }
+    })
```

Comment:
can we make use of the update function here instead of tapping directly into the store? and maybe we can filter things out?